scripts/killApp.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
You should add a comment to the top of this file stating that this script should only be run locally and never on CI.